libs/common-components/src/lib/modal/modal.component.html

```diff
@@ -1,5 +1,6 @@
 <div class="gd-modal fade" id="modalDialog" (click)="onClose($event);" *ngIf="visibility">
-  <div class="gd-modal-dialog">
+</div>
```

Comment:
Changes here and below according to the second option from https://tympanus.net/codrops/2013/11/07/css-overlay-techniques/.